nx/include/switch/arm/cache.h

```diff
@@ -1,7 +1,37 @@
+/**
+ * @file cache.h
+ * @brief AArch64 cache operations.
+ * @author plutoo
+ * @copyright libnx Authors
+ */
 #pragma once
+#include "../types.h"
 
+/**
+ * @brief Performs a data cache flush on the specified buffer.
+ * @param addr Address of the buffer.
```

Comment:
Beware -- addr, size will be forcibly aligned to cache line size in all four functions.